CareKitFHIR/CareKitFHIR.xcodeproj/project.pbxproj

```diff
@@ -22,12 +22,13 @@
 		03AFA931233E69830091DD45 /* OCKR4PatientCoder.swift in Sources */ = {isa = PBXBuildFile; fileRef = 03AFA92E233E69830091DD45 /* OCKR4PatientCoder.swift */; };
 		03AFA93D233E74610091DD45 /* FHIRResourceDataTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 03AFA93C233E74610091DD45 /* FHIRResourceDataTests.swift */; };
 		03AFA943233E80C40091DD45 /* DSTU2PatientConverterTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 03AFA942233E80C40091DD45 /* DSTU2PatientConverterTests.swift */; };
-		03C75C5724A50248009F1896 /* ModelsR4 in Frameworks */ = {isa = PBXBuildFile; productRef = 03C75C5624A50248009F1896 /* ModelsR4 */; };
-		03C75C5924A50248009F1896 /* ModelsDSTU2 in Frameworks */ = {isa = PBXBuildFile; productRef = 03C75C5824A50248009F1896 /* ModelsDSTU2 */; };
 		03C85E6E233D71EC002C5901 /* Utils.swift in Sources */ = {isa = PBXBuildFile; fileRef = 03C85E6D233D71EC002C5901 /* Utils.swift */; };
 		03C85E73233D7215002C5901 /* CareKitFHIRTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 03C85E72233D7215002C5901 /* CareKitFHIRTests.swift */; };
 		03C85E76233D721F002C5901 /* R4PatientConverterTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 03C85E75233D721F002C5901 /* R4PatientConverterTests.swift */; };
 		03E5929123736D0D00B6E9DE /* DSTU2TaskConverterTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 03E5929023736D0D00B6E9DE /* DSTU2TaskConverterTests.swift */; };
+		915266B424A7C95E009CB68A /* ModelsDSTU2 in Frameworks */ = {isa = PBXBuildFile; productRef = 915266B324A7C95E009CB68A /* ModelsDSTU2 */; };
+		915266B624A7C95E009CB68A /* ModelsR4 in Frameworks */ = {isa = PBXBuildFile; productRef = 915266B524A7C95E009CB68A /* ModelsR4 */; };
+		915266B824A7C95E009CB68A /* ModelsBuild in Frameworks */ = {isa = PBXBuildFile; productRef = 915266B724A7C95E009CB68A /* ModelsBuild */; };
```

Comment:
Did you add the `ModelsBuild` framework because of build errors on Travis, or was this an unintentional change?

Reply:
This was unintentional, as I just selected all of the packages when switching to https. I removed the extra one